website/static/js/fangorn.js

```diff
@@ -207,10 +207,10 @@ function _fangornUploadProgress(treebeard, file, progress) {
     } else {
         column = 1;
     }
-    msgText  += 'Uploaded ' + Math.floor(progress) + '%'
+    msgText  += 'Uploaded ' + Math.floor(progress) + '%';
 
     if (progress < 100) {
-        item.notify.update(msgText, 'success', column, 0);
+        item.notify.update(msgText, 'success', column, 1);
```

Comment:
While this works, it is a hack. The problem is here: https://github.com/caneruguz/treebeard/blob/master/scripts/grid.js#L151 ; if `timeout` is 0, the `Notify's` timeout will be set to `3000`.

Can you make a quick fix to treebeard?